container/features/pom.xml

```diff
@@ -235,6 +235,7 @@
             <feature>opennms-activemq-event-forwarder</feature>
             <feature>opennms-activemq-event-receiver</feature>
             -->
+            <feature>opennms-activemq-shell</feature>
```

Comment:
Feature is added, but not installed by default to avoid changing configuration files. Can be installed ad-hoc when needed.